spec/footer_spec.rb

```diff
@@ -8,6 +9,7 @@
     @header.click_my_account
```

Comment:
remove the header class instance as it is not being used here and with the new login will not be used too.